timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -651,38 +685,43 @@ export default {
       leftDrawer: true,
       expandedRows: [],
       timeFilterMenu: false,
-      addManualEvent: false,
+      selectedFields: [{ field: 'message', type: 'text' }],
+      searchColumns: '',
+      columnDialog: false,
       saveSearchMenu: false,
       saveSearchFormName: '',
       fromSavedSearch: false,
+      selectedEventTags: [],
+      showRightSidePanel: false,
+      addManualEvent: false,
       datetimeManualEvent: '', // datetime of an event used
+      // TODO: Refactor this into a configurable option
```

Comment:
Do we have an issue that we can link to this todo?

Reply:
Good call - I created an issue and linked to it in the comment.